extra/3-magic-8-ball.js

```diff
@@ -45,8 +45,36 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+
+  const answers = [
+    ["It is certain.", "very positive"],
+    ["It is decidedly so.", "very positive"],
+    ["Without a doubt.", "very positive"],
+    ["Yes - definitely.", "very positive"],
+    ["You may rely on it.", "very positive"],
+    ["As I see it, yes.", "positive"],
+    ["Most likely.", "positive"],
+    ["Outlook good.", "positive"],
+    ["Yes.", "positive"],
+    ["Signs point to yes.", "positive"],
+    ["Reply hazy, try again.", "negative"],
+    ["Ask again later.", "negative"],
+    ["Better not tell you now.", "negative"],
+    ["Cannot predict now.", "negative"],
+    ["Concentrate and ask again.", "negative"],
+    ["Don't count on it.", "very negative"],
+    ["My reply is no.", "very negative"],
+    ["My sources say no.", "very negative"],
+    ["Outlook not so good.", "very negative"],
+    ["Very doubtful.", "very negative"],
+  ];
+
 function shakeBall() {
-  //Write your code in here
+  let randomNumber = Math.trunc(Math.random() * answers.length);
+  let randomAnswer = answers[randomNumber][0];
```

Comment:
Impressive use of a "2D array"

Reply:
These have been changed after looking at the solutions folder (for the review).